javascript/react/src/components/ArqueroDetailsList/hooks/useStripedRowsRenderer.tsx

```diff
@@ -17,21 +18,29 @@ export function useStripedRowsRenderer(
 	striped: boolean,
 	columnBorders: boolean,
 	compactRowHeight: number,
+	hideRowNumber?: boolean,
 ): IRenderFunction<IDetailsRowProps> {
 	return useCallback(
 		(props: IDetailsRowProps | undefined) => {
 			if (!props) {
 				return null
 			}
+			if (!hideRowNumber) {
+				props.item = {
```

Comment:
I wonder if this could be done entirely virtually with a custom RowNumberColumnRenderer that is assigned to the onRender